webapp/src/css/inbox.less

```diff
@@ -108,6 +109,53 @@ body {
   display: none;
 }
 
+#privacy-policy-wrapper {
+  z-index: 11;
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  overflow: hidden;
+  background: white;
+
+  .container {
+    height: 100%;
+
+    .loader {
+      top: 50%;
+      margin-top: -20px;
+    }
+
+    .content {
+      height: 100%;
+      padding-bottom: 67px;
+      position: relative;
+
+      .html-content {
+        height: 100%;
+        overflow-y: auto;
+      }
+
+      .footer {
+        position: absolute;
+        border-top: 1px solid @gray-light;
```

Comment:
Don't use `@gray-light`. There's a warning in the first line of the `varaibles.less` file:

> /* Basic colors. WARNING: Don't use these outside this file as the variable names are not useful */

`@separator-color` is probably what you're after.

Reply:
Done.